Cargo.toml

```diff
@@ -60,7 +60,7 @@ indexmap = "2.2.6"
 itertools = "0.12.1"
 jobserver = "0.1.31"
 lazycell = "1.3.0"
-libc = "0.2.154"
+libc = "0.2.155"
```

Comment:
Why is libc being bumped as part of this PR?

Reply:
I've removed it. (It had something to do with 0.2.154 being yanked, but I don't remember anymore why that was a problem)